pkgs/development/ocaml-modules/janestreet/janePackage_0_13.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, fetchFromGitHub, buildDunePackage, defaultVersion ? "0.13.0" }:
+
+{ pname, version ? defaultVersion, hash, ...}@args:
+
+buildDunePackage (args // {
+  inherit version;
+
+  minimumOCamlVersion = "4.08";
+
+  src = fetchFromGitHub {
```

Comment:
janestreet now also has dune release tarballs, i. e. `https://github.com/janestreet/${pname}/releases/download/v${version}/${pname}-v${version}.tbz`, maybe its worth it to use them via `fetchurl`?

Reply:
Who will update all the hashes?

Reply:
Oh right, probably not worth it...